MISampleApp/MISampleApp/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="MISampleAppType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+  <Parameters>
+    <Parameter Name="MISampleConsole_InstanceCount" DefaultValue="-1" />
```

Comment:
This makes the service run on every node in the cluster. That makes it more likely to hit a throttling condition. Did you intend to have this sample run as a -1 service? (If so, why?)